web/modules/custom/do_base/do_base.deploy.php

```
@@ -8,3 +8,331 @@
+
+    $node = $node_storage->create($values);
+    $node->save();
+  }
+  catch (\Throwable $throwable) {
+    $transaction->rollBack();
+
+    throw $throwable;
+  }
+
+  return sprintf('Created the How We Work page (node %s).', $node->id());
+}
```

PHPMD (2.15.0)

[warning] The function do_base_deploy_populate_how_we_work_page() has 321 lines of code. Current threshold is set to 100. Avoid really long methods. (undefined)

(ExcessiveMethodLength)